packages/yew/src/scheduler.rs

```diff
@@ -292,7 +218,12 @@ impl Scheduler {
         if !to_run.is_empty() {
             return;
         }
-        to_run.extend(self.rendered_first.drain(..).rev());
+
+        if !self.rendered_first.is_empty() {
+            let mut rendered_first = BTreeMap::new();
+            std::mem::swap(&mut self.rendered_first, &mut rendered_first);
```

Comment:
```suggestion
            let rendered_first = std::mem::take(&mut self.rendered_first);
```
Also a second time later on.